driver.go

```diff
@@ -123,8 +124,16 @@ func (d *sqlDriver) unlock() {
 	d.mu.Unlock()
 }
 
-// Open returns a new connection to the database.  The name is a string in a
-// driver-specific format.
+// Open returns a new connection to the database.
+//
+// The dataSource must be formatted as an URL 'engine://path/dbname?params'.
+// Engine is the storage name registered with RegisterStore.
+// Path is the storage specific format.
+// Params is key-value pairs split by '&', optional params are storage specific.
+// Examples:
+//    goleveldb://relative/path/test
+//    boltdb:///absolute/path/test
+//    hbase://localhost,zkhost1,zkhost2/test?tso=127.0.0.1:1234&table=tidb
```

Comment:
@coocood  table is for hbase table, not TiDB database table. 
So I think we should use a better understanding name.